examples/base/src/Router.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Router, Route, Text, useRouter, ButtonGroup, Button } from '@urban-bot/core';
+
+function Route1() {
```

Comment:
It is not very important, but I think "Route1" is tautological.
``` tsx
   <Router>
            <Route path="/">
                  <Route1 />
```

Should you use a more meaningful name, such as  "WelcomeMessage" or "Profile".

Reply:
fixed